EliteDangerous/JournalEvents/translation-example-je.tlp

```diff
@@ -861,6 +861,17 @@ SECTION ExobiologyClass
 .OrganicGenus: "Unique Genus Data Logged: ;;N0" @
 .OrganicSpecies: "Unique Species Data Logged: ;;N0" @
 
+SECTION ThargoidsClass
+
+.EncounterWakes: "Thargoid wakes scanned: ;;N0" @
+.EncounterKilled, "Thargoids killed: ;;N0" @
+.EncounterImprint, "Thargoid structures: ;;N0" @
+.EncounterTotal, "Total encounters: ;;N0" @
```

Comment:
Colon not comma

Reply:
Stupid me copying things without changing it properly, sorry.
Wonder why I stopped after the first...